### ISSUE TITLE
Dataverse "pretty url" and "regular url"  give different search results (48 results vs. 75k+ results)

### ISSUE THREAD

**Issue by raprasad** [IQSS/dataverse#3600]
Discovered this when clicking from "mydata" which is using the non-pretty url.  (MyData links should be changed regardless of this bug)

---

## to do

- [ ] fix dataverse page filtering even when using parameterized url (e.g. dataverse id vs alias)
- [ ] update mydata links to dataverses to use the alias

![gary_king_dataverse_and_gary_king_dataverse_and_iqss_geoconnect_and_coffee_-_root_dataverse](https://cloud.githubusercontent.com/assets/1036060/22375609/d8600040-e478-11e6-85a0-a0927de3384e.png)

**Comment by pdurbin** [comment]
@raprasad huh. Are you saying you can get to the non-pretty URL by clicking something from the GUI? If so, where?

**Comment by raprasad** [comment]
mydata "cards_minimum.html" has a link to a parent dataverse that's "not pretty"

To fix this, additional data is needed in the JSON for ```card_info.parentAlias```

```html
 {% if card_info.parentId %}
    {% if card_info.parentId != card_info.identifier %}
         -  <a href="/dataverse.xhtml?id={{ card_info.parentId }}">{{ card_info.parentName }} Dataverse</a>
    {% endif %}
 {% endif %}
```

**Comment by scolapasta** [comment]
the issue with the non pretty url has to do with the search include fragment not being aware of the dataverse id.

**Comment by scolapasta** [comment]
FYI, there is also a nonpretty version of the url that uses alias (what the pretty url maps too). That should work fine. So if you do need to use non pretty urls somewhere, use those.

**Comment by mheppler** [comment]
These links from My Data have been fixed since 4.8.2 fcec2dcd0cf76671742f11d41b5d3f1da5acb71d, but as @scolapasta suggests, logic should be added to the search-include to resolve the incorrect counts/results.

**Comment by cmbz** [comment]
To focus on the most important features and bugs,     we are closing issues created before 2020 (version 5.0)     that are not new feature requests with the label 'Type: Feature'.    

If you created this issue and you feel the team     should revisit this decision, please reopen the issue and leave a comment.
